ports/nrf/README.md

```diff
@@ -114,13 +116,13 @@ Install the necessary tools to flash and debug using Segger:
 
 [JLink Download](https://www.segger.com/downloads/jlink#)
 
-[nrfjprog linux-32bit Download](https://www.nordicsemi.com/eng/nordic/download_resource/52615/16/95882111/97746)
+[nrfjprog linux-32bit Download](https://www.nordicsemi.com/eng/nordic/download_resource/52621/24/48793639/97746)
```

Comment:
These changes should really have been done in a separate PR, as they're unrelated to this board.

Reply:
I merely updated the current ones with the new ones, but as this is a manual process pointing to the pages with the overview of the current and previous tools is even better.